probe-dictionary/explorer.css

```diff
@@ -122,3 +122,15 @@ body {
 .hidden {
     display: none;
 }
+
+.contrast:after {
+    content: "";
```

Comment:
Do we need this change?
We should probably be able to just use `background-color`, `margin` and `padding` on this element.
Let's see how that works, we might need to adjust the surrounding elements a little too?

Reply:
@georgf Ya actually I had tried that, but a default margin was set up on the page ,so I used html { margin :0} but the background color still didn't extend beyond the margin. Hence I wrote this...I'll still try using margin and padding in the code!